src/www/firewall_rules.php

```diff
@@ -477,6 +489,28 @@
                         $selected_if == 'FloatingRules'
                      )
                 ):
+				  if( $selected_if != $interface )
+                  {
+					$offset = $i;
+                    $interface = $selected_if;
+                  }
+                  $idx = $i-$offset;
```

Comment:
I'm not sure what this is supposed to be doing, if we only want to add a number to the rule, this feels a bit too much.

Reply:
Numbering rules per interface helps addressing the *same* rule in a discussion.
I'm not aware of an easier way to get the number.
The rules data structure is not per interface, therefore I choose to store the interfaces offset.